README.md

```diff
@@ -82,3 +82,66 @@ PING dns.docker (172.18.0.30) 56(84) bytes of data.
 3 packets transmitted, 3 received, 0% packet loss, time 2001ms
 rtt min/avg/max/mdev = 0.025/0.038/0.049/0.011 ms
 ```
+
+## swarm support
+`rawdns` can be used with swarm by creating a configuration that provides the socket details using the `tcp://` schemem.  You will also need to enable `swarmnode` by setting it to true.  The `swarmnode` option enables `rawdns` to look at the `Node` section of the inspect API response for the external/host IP address.
```

Comment:
Can you please add a single blank line between the header and the paragraph?  Also, only a single blank line following the paragraph and `s/schemem/scheme/`.